crmsh/config.py

```diff
@@ -303,6 +303,17 @@ def __init__(self):
         self._systemwide = None
         self._user = None
 
+    def _safe_read(self, config_parser_inst, file_list):
+        """
+        Try to handle configparser.MissingSectionHeaderError while reading
+        """
+        from . import utils
+        try:
+            config_parser_inst.read(file_list)
+        except configparser.MissingSectionHeaderError as e:
+            with utils.disable_exception_traceback():
+                raise ValueError(e) from None
```

Comment:
I think it would be more logical to raise `MissingSectionHeaderError` instead of `ValueError`. The 1st one is more meaningful

Reply:
Changed